### PR TITLE
ped.ParachuteTo()

### PR DESCRIPTION
**Pull request by Tanjitsu**

### CODE REVIEW

File: source/scripting/Ped.cs [hunk 1]
```diff
@@ -1555,6 +1555,10 @@ public void OpenParachute()
 		{
 			Function.Call(Hash.FORCE_PED_TO_OPEN_PARACHUTE, Handle);
 		}
+		public void ParachuteTo(Vector3 location)
+		{
+		    	Function.Call(Hash.SET_PARACHUTE_TASK_TARGET, Handle, location.X, location.Y, location.Z);
```
Comment (crosire) [inline source/scripting/Ped.cs]: Isn't there https://github.com/crosire/scripthookvdotnet/blob/dev_v3/source/scripting/Tasks.cs#L249 already which does exactly this?

Comment (Tanjitsu) [inline source/scripting/Ped.cs]: Should i name it differently... ?
This is to be used outside of a Task sequence, to update the position of the task.
Such as....
new Sequence()->Tasks.ParachuteTo(moving.entity.position)->PerformSequence();

OnTick->If Sequence Active->Ped.ParachuteTo(moving.entity.position);

Then the task sequence is uninterrupted but the ped's parachute target is updated per the moving targets position.

Comment (Tanjitsu) [inline source/scripting/Ped.cs]: That sets a new Task though and breaks the current sequence.

Comment (crosire) [inline source/scripting/Ped.cs]: It does nothing but call the exact same native, so I don't see why it would behave any different?

Comment (crosire) [inline source/scripting/Ped.cs]: I see. Didn't realize the two call different natives.

**Comment by Tanjitsu** [comment]
I have pull requests queued waiting on my open pull request.
How long does it take to get a pull request accept or decline ?

#649 opened 22 days ago by Tanjitsu
All i added was a comment and a "new" method.

**Pull request merged**